app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -430,10 +430,6 @@ class GleanMetricsService(private val context: Context) : MetricsService {
         }
     }
 
-    /**
-     * Sets some basic Glean metrics required by Fenix.
-     * This is a separate function to simplify testing.
```

Comment:
Does this comment need to be removed?

Reply:
@NotWoods There was a nit in the original PR that questioned if we needed this comment.